app/liquid_tags/github_tag/github_issue_tag.rb

```diff
@@ -68,7 +85,19 @@ def valid_link?(link)
     end
 
     def raise_error
-      raise StandardError, "Invalid Github issue link"
+      raise StandardError, "Invalid GitHub issue, pull request or comment link"
+    end
+
+    def title
+      content_json[:title] || "Comment for"
```

Comment:
comments don't have a `title` attribute, so we add a verbatim title, see the screenshots